setup.py

```diff
@@ -1,8 +1,47 @@
 # -*- coding: utf-8 -*-
 
 from setuptools import setup
+from pathlib import Path
+import shutil
 import versioneer
 
+
+def copy_mda_assets():
+    static_dir = Path('.') / 'mdanalysis_sphinx_theme' / 'static'
+    branding_dir = Path('.') / 'branding'
+
+    # outdir for logos and icons
+    static_logos = static_dir / 'logo'
+
+    # main logo to appear in nav bar
+    mda_logo: Path = branding_dir / 'logos' /\
+                                    'rastered' /\
+                                    'mdanalysis-logo_bgwhite@600ppi.png'
```

Comment:
```suggestion
    mda_logo_base_file = "mdanalysis-logo_bgwhite@600ppi.png"
    mda_logo: Path = branding_dir / 'logos' / 'rastered' / mda_logo_base_file
```

Could we maybe get this path onto one line with the above workaround, or maybe encapsulate it in parentheses? In general PEP8 prefers shorter lines, or wrapping code in parentheses, over backslash-style line continuation.